react-frontend/src/components/StyleComponents/headings.js

```diff
@@ -8,6 +8,25 @@ export const CaseStudyHeading = styled.h2.attrs({
   margin-bottom: 24px;
 `;
 
+export const DigitalFrameworkHeading = styled.h2.attrs({
+  className: 'digitalFrameworkHeading',
+})`
+  font-size: 22pt;
+  font-weight: bold;
+  line-height: 24pt;
+  margin-bottom: 5px;
+`;
+
+export const DigitalFrameworkSubHeading = styled.h3.attrs({
+  className: 'digitalFrameworkSubHeading',
+})`
+  font-size: 18pt;
+  font-weight: bold;
+  line-height: 24pt;
+  margin-bottom: 5px;
+  margin-top: 5px;
+`;
```

Comment:
Can we use style extensions here? 
https://styled-components.com/docs/basics#extending-styles

Reply:
good idea, I was so focused on pulling things into styled components I didn't try simplifying them once they'd been moved. 
Implented a few, but I will probably have to do more once the rest of the css file have been removed. (I'll probably need to do the same with page content to make alignment a bit more robust.)